en/clubs/ovriga_funktionarer/body.md

```diff
@@ -36,9 +36,11 @@ Responsible for the chapter's communication channels, like Discord, Slack, Faceb
 The program responsible student works together with the chairman of the Study Committee to improve the quality of the education. In practice, the work consists of meetings with teachers and those responsible for the program at EECS and meetings centrally at THS.
 
 
-## Sound and light manager
+## Sound- and Lightmanager - Karl-Isac Åström
```

Comment:
```suggestion
## Sound and Light manager - Karl-Isac Åström
```
Pretty sure the previous spelling is more correct in English, maybe even 
```suggestion
## Sound and Lights manager - Karl-Isac Åström
```
is more desirable